### DISCUSSION TITLE
cdn-cname-cloudflare/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#111] [Giscus]
# cdn-cname-cloudflare/

本文介绍了如何实现图床 CDN 域名国内境外分流。作者首先提出了实现分流的目的,即削减成本、提高性能。然后说明了实现思路,即国内域名 CNAME 指向 Cloudflare 作为回源,再通过Cloudflare Worker 访问 R2 或 B2 存储。作者列出了各服务的免费限额,以小网站为例说明国内外分流的具体配置步骤,包括 R2 绑定域名、Cloudflare for SaaS 接入、添加 CNAME 记录、创建 Worker 等。文末总结了实现分流的关键步骤。

https://www.eallion.com/cdn-cname-cloudflare/

**Comment by HowieHz** [comment]
不错不错，mark了

**Reply by eallion** [reply]
谢谢！
